### PR TITLE
feat: add team admin func to update member roles

### PR DESCRIPTION
**Pull request by junminahn**
- Assign team admin to update team members' roles.

### CODE REVIEW

File: lambda/app/src/controllers/team.ts [hunk 1]
```diff
@@ -174,3 +175,19 @@ export const removeUserFromTeam = async (userId: number, teamId: number) => {
...
+      where: {
+        userId,
+        teamId,
+      },
+      returning: true,
+      plain: true,
+    },
+  );
+
+  return getMmberOnTeam(teamId, userId, { raw: true });
```
Comment (thegentlemanphysicist) [inline lambda/app/src/controllers/team.ts:192]: spelling error on name

File: lambda/app/src/queries/team.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { sequelize, models } from '../../../shared/sequelize/models/models';
+
+export const getMmberOnTeam = async (teamId: number, userId: number, options: { raw: boolean }) => {
```
Comment (thegentlemanphysicist) [inline lambda/app/src/queries/team.ts:3]: spelling on Member

**Review by thegentlemanphysicist — COMMENTED** [review]
Noticed a typo in one of the method names.
Also I noticed the id was being changed from a string to the date, do we need to migrate existing ids in the DB for this?

File: lambda/app/src/controllers/team.ts (hunk 1 above)
Comment (junminahn) [reply lambda/app/src/controllers/team.ts]: nice cache! fixed it

File: lambda/app/src/queries/team.ts (hunk 2 above)
Comment (junminahn) [reply lambda/app/src/queries/team.ts]: nice cache! fixed it

**Review by thegentlemanphysicist — APPROVED** [review]

**Pull request merged**